core/src/main/java/cz/xtf/core/bm/BuildManager.java

```diff
@@ -16,6 +17,20 @@ public BuildManager(OpenShift openShift) {
         if (openShift.getProject(openShift.getNamespace()) == null) {
             openShift.createProjectRequest();
             openShift.waiters().isProjectReady().waitFor();
+
+            OpenShift adminOcp = OpenShifts.admin();
+            if (adminOcp != null) {
+                // Adding a label can be only done via 'namespace'. It cannot be set via 'project' API.
+                // Thus we do this separately. Also, to update namespace label, it's necessary to have 'cluster-admin'
+                // role binding for current user and updated namespace. Otherwise you can see:
+                // $ oc label namespace <name> "label1=foo"
+                // Error from server (Forbidden): namespaces "<name>" is forbidden: User "<user>" cannot patch resource "namespaces" in API group "" in the namespace "<name>"
+                OpenShifts.admin().namespaces().withName(openShift.getNamespace()).edit().editMetadata().addToLabels(
```

Comment:
Wouldn't be safest try this command, and catch exception and not fail on that?

Reply:
Probably yes... it will be easier at least... my first intention was to extend the `OpenShifts` class to determine somehow whether we are able to gain admin account and privileges... but yeah, I guess I'm too lazy to do that now :)